melodies_monet/plots/surfplots.py

```diff
@@ -106,12 +109,34 @@ def new_color_map():
         Orange and blue color map
         
     """
-    top = mpl.cm.get_cmap('Blues_r', 128)
-    bottom = mpl.cm.get_cmap('Oranges', 128)
+    top = mpl.cm.get_cmap("Blues_r", 128)
+    bottom = mpl.cm.get_cmap("Oranges", 128)
+
     newcolors = np.vstack((top(np.linspace(0, 1, 128)),
                            bottom(np.linspace(0, 1, 128))))
     return ListedColormap(newcolors, name='OrangeBlue')
 
+def new_color_map2():
+    """
+    # JianHe: create new colors here, fix for mpl v3.6+
+    Creates new color map for difference plots
+       
+    Returns
+    -------
+    colormap
+        Orange and blue color map
+
+    """
+    top = mpl.colormaps['Blues_r'].resampled(128)
+    bottom = mpl.colormaps['Oranges'].resampled(128)
+
+    newcolors = np.vstack((top(np.linspace(0, 1, 128)),
+                           bottom(np.linspace(0, 1, 128))))
+    newcmap = ListedColormap(newcolors, name='OrangeBlue')
+    mpl.colormaps.register(newcmap)
+
+    return newcmap
```

Comment:
I think instead of this, we should just register the colormap (if not already registered) near the top of this file.

Then the `try` blocks are not needed.